docs/get-started/lotus/installation.md

```diff
@@ -278,14 +295,20 @@ We recommend that macOS users use [Homebrew](https://brew.sh) to install each of
    brew install go bzr jq pkg-config rustup hwloc
    ```
 
+2. Make sure all the packages are installed successfully before you move to next step to build lotus.
+
+### Build and install Lotus
+
+Once all the dependencies are installed, you can build and install the Lotus suite (`lotus`, `lotus-miner`, and `lotus-worker`).
+
 1. Clone the repository:
 
    ```sh
    git clone https://github.com/filecoin-project/lotus.git
    cd lotus/
    ```
 
-1. To join mainnet, checkout the [latest release](https://github.com/filecoin-project/lotus/releases).
+2. To join mainnet, checkout the [latest release](https://github.com/filecoin-project/lotus/releases).
```

Comment:
```suggestion
1. Checkout the release for the network you wish to use.

To join mainnet, checkout the [latest release](https://github.com/filecoin-project/lotus/releases). For networks other than mainnet...
```

Also move text from line 315 to the end of the line above. This introduces the idea of different networks first, as you suggested in our chat.